client/components/pages/configuration/general/DefaultDiskQuota.vue

```diff
@@ -0,0 +1,94 @@
+<script setup lang="ts">
+const {
+  quotaPolicyValue,
+  quotaValue,
+} = defineProps<{
+  quotaPolicyValue: DiskQuotaPolicy
+  quotaValue: number | null
+}>()
+const quotaPolicy = ref(quotaPolicyValue)
+const quota = ref<number | null>(quotaValue)
+
+const { t } = useI18n()
+const { patchConfiguration, isError } = useConfigurationFields({
+  defaultDiskQuotaPolicy: quotaPolicy,
+  defaultDiskQuota: quota,
+}, ({ defaultDiskQuotaPolicy, defaultDiskQuota }) => ({
+  defaultDiskQuotaPolicy: defaultDiskQuotaPolicy.value,
+  defaultDiskQuota: defaultDiskQuota.value,
+}))
+
+const options = computed(() => [
+  { icon: 'i-tabler-infinity', name: t('pages.configuration.general.default_disk_quota.unlimited'), value: 'UNLIMITED' },
+  { icon: 'i-tabler-database', name: t('pages.configuration.general.default_disk_quota.limited'), value: 'LIMITED' },
+])
+const inputPlaceholder = computed(() => {
+  if (quotaPolicy.value === 'UNLIMITED') return t('unlimited_quota')
+  return t('pages.configuration.general.default_disk_quota.placeholder')
+})
+
+function getOption(value: string) {
+  return options.value.find(option => option.value === value)
+}
+
+function patch() {
+  if (quotaPolicy.value === 'UNLIMITED') quota.value = null
+  if (quotaPolicy.value === 'LIMITED' && quota.value === null) return
+
+  patchConfiguration({
+    defaultDiskQuotaPolicy: quotaPolicy,
+    defaultDiskQuota: quota,
+  })
+}
+
+watch(quotaPolicy, () => {
+  if (quotaPolicy.value === 'UNLIMITED') quota.value = null
+})
+</script>
+
+<template>
+  <HorizontalActionPanel
+    :header="t('pages.configuration.general.default_disk_quota.title')"
+    :description="t('pages.configuration.general.default_disk_quota.description')"
+    :error="isError"
+  >
+    <div class="flex flex-col gap-2">
+      <PDropdown
+        v-model="quotaPolicy"
+        :options="options"
+        option-label="name"
+        option-value="value"
+        class="w-full md:w-14rem"
+        :class="{ 'p-invalid': isError }"
+        @update:model-value="patch"
+      >
```

Comment:
```suggestion
      <PDropdown
        :options="options"
        option-label="name"
        option-value="value"
        class="w-full md:w-14rem"
        :class="{ 'p-invalid': isError }"
        :model-value="quotaPolicy"
        @update:model-value="patch"
      >
```

don't duplicate v-model expanded properties (v-model = :model-value + @update:model-value)